services/hanke-service/src/main/resources/application.yml

```diff
@@ -1,4 +1,9 @@
 haitaton:
+  ad:
+    filter:
+      use: ${HAITATON_USE_AD_FILTER:false}
+      # List of allowed groups, seperated by ;
+      allowed-groups: ${HAITATON_ALLOWED_AD_GROUPS:}
```

Comment:
Should there also be envs for these new properties in docker-compose.yml for haitaton-hanke:
```
      HAITATON_USE_AD_FILTER: ${HAITATON_USE_AD_FILTER:-false}
      HAITATON_ALLOWED_AD_GROUPS: ${HAITATON_ALLOWED_AD_GROUPS:-}
```

Reply:
Adding those would allow us to modify the values without rebuilding the backend service, but I don't think we'll need to change these. We don't have the required AD groups anyway.